DARNprocessing/plotting_scripts/convectionmaps.py

```diff
@@ -90,271 +99,304 @@ def __init__(self, arguements=None, parameters=None):
         else:
             self.parameter = {'date': None,
                               'channel': 5,
+                              'integration_time': 120,
                               'hemisphere': 'north',
                               'start_time': '00:00',
                               'end_time': '23:59',
                               'rst_version': 4.1,
-                              'image_extension': 'pdf',
-                              'logfile': self._current_path+'/ConvectionMaps.log',
-                              'datapath': self._current_path,
-                              'plotpath': self._current_path,
-                              'omnipath': self._current_path,
-                              'mappath': self._current_path,
-                              'keypath': self._current_path,
+                              'image_ext': 'pdf',
+                              'logfile': self._current_path +
+                              '/ConvectionMaps.log',
+                              'data_path': self._current_path,
+                              'plot_path': self._current_path,
+                              'map_path': self._current_path,
+                              'key_path': self._current_path,
                               'num_proc': 1}
 
             self.parameter.update(parameters)
+            # Required field
             if not self.parameter['date']:
                 raise ValueError("Date of the date was not passed in, please"
```

Comment:
> "Date of the date"

"Date of the file(s)"? Would separate year, month, and day fields be better? Would be more understandable